helm/cluster-operator/values.yaml

```
@@ -1,4 +1,32 @@
-namespace: giantswarm
-
-userID: 1000
-groupID: 1000
+image:
+  registry: "quay.io"
+  repository: "giantswarm/cluster-operator"
```

```suggestion
  name: "giantswarm/cluster-operator"
```

I'd like to use the shorter name is possible. https://github.com/giantswarm/fmt/blob/master/helm/image_templating.md

---

Done.